packages/mux-player/test/template.test.js

```diff
@@ -19,9 +19,10 @@ describe('<mux-player> template render', () => {
     );
   });
 
-  it('template live extra-small', function () {
+  it('template live extra-small w/o src, not live', function () {
```

Comment:
nitpick(blocking) lol maybe make this a little more verbose, since I'm not sure what it means to be "template live... not live" 😅

Reply:
oops, the second "live" is "not in live window". I'll update it.